### PR TITLE
Minor cleanup on color dialog

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

File: synfig-studio/src/gui/widgets/widget_coloredit.cpp [hunk 1]
```diff
@@ -132,44 +123,33 @@ ColorSlider::draw_arrow(
...
-	//!TODO FActorize ! (Duplicate code with "Widget_Keyframe_List::draw_arrow")
-	//! Upper black pointing down arrow
+	// Upper black pointing down arrow
 	cr->set_source_rgb(dark.get_r(), dark.get_g(), dark.get_b());
 	cr->set_line_width(1.0);
 	cr->move_to(x, y);
 	cr->line_to(x - 0.5*width, y - height);
 	cr->line_to(x + 0.5*width, y - height);
 	cr->close_path();
 	if (fill)
```
Comment (ice0) [inline synfig-studio/src/gui/widgets/widget_coloredit.cpp:137]: Change this to !fill?

File: synfig-studio/src/gui/widgets/widget_coloredit.cpp [hunk 2]
```diff
@@ -132,44 +123,33 @@ ColorSlider::draw_arrow(
...
 
-	//! Bottom light pointing up arrow
+	// Bottom light pointing up arrow
 	cr->set_source_rgb(light.get_r(), light.get_g(), light.get_b());
 	cr->set_line_width(1.0);
 	cr->move_to(x, size - height);
 	cr->line_to(x - 0.5*width, size);
 	cr->line_to(x + 0.5*width, size);
 	cr->close_path();
 	if (fill)
```
Comment (ice0) [inline synfig-studio/src/gui/widgets/widget_coloredit.cpp:149]: !fill?

**Comment by rodolforg** [comment]
Is that what you wanted @ice0 ?

**Comment by ice0** [comment]
Hi! Sorry, i missed this piece of code:
```
cr->fill();
```
and for me it looked like:
```
if (fill) {
}
else cr->stroke();
```
That is why I suggested this change. But now I see it's wrong.

Can you drop last commit and push it again?

**Comment by rodolforg** [comment]
Done

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**